src/js/providers/bandwidth-monitor.ts

```diff
@@ -18,11 +19,17 @@ export default function BandwidthMonitor(provider, initialEstimate) {
                 });
             }, 1000);
         },
-        stop() {
+        stop(): void {
             clearInterval(bandwidthMonitorInterval);
         },
-        getEstimate() {
+        getEstimate(): number {
             return bandwidthEstimate;
         }
     };
 }
+
+export type BandwidthMonitor = {
```

Comment:
this is a personal style thing so up to you, but i would hoist these to the top of the file for ease of access